activerecord/test/cases/adapters/sqlite3/sqlite3_adapter_test.rb

```diff
@@ -238,6 +242,20 @@ def test_insert_id_value_returned
         end
       end
 
+      def test_exec_insert_with_returning_disabled
+        connection = connection_without_insert_returning
+        result = connection.exec_insert("insert into '1_need_quoting' (name) VALUES ('foo')", nil, [], "id")
+        expect = connection.query("select max(id) from '1_need_quoting'").first.first
+        assert_equal expect.to_i, result.rows.first.first
+      end
+
+      def test_exec_insert_default_values_with_returning_disabled
+        connection = connection_without_insert_returning
+        result = connection.exec_insert("insert into '1_need_quoting' DEFAULT VALUES", nil, [], "id")
+        expect = connection.query("select max(id) from '1_need_quoting'").first.first
+        assert_equal expect.to_i, result.rows.first.first
+      end
```

Comment:
This test suite is using an in-memory database, so unfortunately we can't rely on the `1_need_quoting` from the test schema here.

I think we can use the `with_example_table` helper to define a test table for use in this test :)

Reply:
Indeed. I saw the test failure and have been fixing. Fix is incoming in mere minutes

Reply:
Fixed